src/nodedef.h

```diff
@@ -35,6 +35,9 @@ class Client;
 #include "constants.h" // BS
 #include "tileanimation.h"
 
+// protocol_version >= 37
+#define CONTENTFEATURES_VERSION 13
```

Comment:
Please use `static const u8` which is more C++ than a define, it's recommended to do this in C++11 guidelines